web/src/components/wizards/S3LogSourceWizard/S3LogSourceWizard.tsx

```diff
@@ -60,6 +62,7 @@ const validationSchema = Yup.object().shape<S3LogSourceWizardValues>({
     )
     .required(),
   kmsKey: Yup.string().matches(getArnRegexForService('KMS'), 'Must be a valid KMS ARN'),
+  managedBucketNotifications: Yup.boolean(),
```

Comment:
It will always be `true` or `false` and never `undefined`

```suggestion
  managedBucketNotifications: Yup.boolean().required(),
```

Reply:
Good call